staging/src/k8s.io/legacy-cloud-providers/azure/clients/loadbalancerclient/mockloadbalancerclient/doc.go

```diff
@@ -1,6 +1,3 @@
-//go:build !providerless
```

Comment:
why remove it? it should not be there, we don't want to build if providerless, do we?

Reply:
will result in an empty package with no go files which makes the compile sad.

Reply:
oh, I meant, all the files under azure folder

Reply:
all the doc.go there should not be there. you can see the effect if you undo these and then run `hack/verify-typecheck-providerless.sh`